### PR TITLE
Added rules to monzo.com (US coverage)

### PR DESCRIPTION
**Pull request by icodebyamanda**
Fixes: https://app.asana.com/0/1202689882543781/1202683416335161/f

### CODE REVIEW

File: rules/autoconsent/monzo-com.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "name": "monzo.com",
+    "prehideSelectors": [".cookie-alert"],
+    "detectCmp": [
+        { 
+            "exists": ".cookie-alert"
+        },
+        {
+            "exists": "[role=\"dialog\"]"
+        }
+],
```
Comment (muodov) [inline rules/autoconsent/monzo-com.json:11]: this looks very generic. It says "there should be an element with cookie-alert AND there should be an element with role="dialog"". But I believe we need them on the SAME element, did you mean `.cookie-alert[role=\"dialog\"]`?

And even like this, we might need an extra check, given that we already had a conflict with another rule, I can easily imagine another page using .cookie-alert with role=dialog. Could you add another check as well? Perhaps something matching the inner structure of the banner?

File: rules/autoconsent/monzo-com.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "monzo.com",
+    "prehideSelectors": [".cookie-alert"],
```
Comment (muodov) [inline rules/autoconsent/monzo-com.json:3]: I think we need to make it more specific too, so we don't occasionally hide unhandled popups

File: lib/config.ts [hunk 3]
```diff
@@ -1 +1 @@
-export const enableLogs = false; // change this to enable debug logs
+export const enableLogs = true; // change this to enable debug logs
```
Comment (muodov) [inline lib/config.ts:1]: please don't forget to change this back before merging

**Review by muodov — CHANGES_REQUESTED** [review]

**Comment by icodebyamanda** [comment]
Made the following chnages following the requests:

1. to increase specificity of `detectCmp`: 
- grouped together .cookie-alert & [role=\"dialog\"] under the same element 
- added `a[href*=\"monzo\"]` as others selectors were too generic and in case this url changes or disappears it'd be easier I think to figure out that's the issue. Also, wanted to make sure that this works on the US & EU websites which are similarly coded.

2. Added another prehideSelectors (no much choice)

3. logs disabled

Besides, regression test are fine (checked 3 times on monzo branch and failures ended up matching those from main).

File: rules/autoconsent/monzo-com.json (hunk 1 above)
Comment (icodebyamanda) [reply rules/autoconsent/monzo-com.json]: see comment below: https://github.com/duckduckgo/autoconsent/pull/22#issuecomment-1204897857

File: rules/autoconsent/monzo-com.json (hunk 2 above)
Comment (icodebyamanda) [reply rules/autoconsent/monzo-com.json]: see comment below: https://github.com/duckduckgo/autoconsent/pull/22#issuecomment-1204897857

File: lib/config.ts (hunk 3 above)
Comment (icodebyamanda) [reply lib/config.ts]: see comment below: https://github.com/duckduckgo/autoconsent/pull/22#issuecomment-1204897857

**Review by muodov — APPROVED** [review]
lgtm!

**Pull request merged**